ci-operator/step-registry/ipi/conf/powervs/ipi-conf-powervs-commands.sh

```diff
@@ -54,7 +54,13 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 # Populate install-config with Powervs Platform specifics
 # Note: we will visit this creation of install-config.yaml file section once the profile support is added to the powervs environment
 POWERVS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.powervscred"
-CLUSTER_NAME="rdr-multiarch-${LEASED_RESOURCE}"
+if [[ -n "${CLUSTER_NAME_MODIFIER}" ]]; then
+  # Hopefully the entire hostname (including the BASE_DOMAIN) is less than 255 bytes.
+  # Also, the CLUSTER_NAME seems to be truncated at 21 characters long.
+  CLUSTER_NAME="p-${LEASED_RESOURCE}-${CLUSTER_NAME_MODIFIER}"
```

Comment:
could we do `rdr-` instead of `p-`? Not trying to be pedantic, it's just that `rdr` is often used to identify what resources belong to our org

Reply:
I know I've mentioned this before, but we have a very small character limit for the cluster name.  `rdr` just digs into that.

Reply:
okay fair enough - we may be asked to switch to `rdr-` at some point but we can see if `p-` is okay for now.